October 2025

buuhuu opened the pull request:
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--{repo}--{owner}.aem.live/
- After: https://<branch>--{repo}--{owner}.aem.live/

devpatil7 committed:
Merge pull request #590 from hlxsites/merge-latest2

Merge latest main to integration

devpatil7 committed:
Merge branch 'main' into update-latest3

devpatil7 committed:
Merge pull request #593 from hlxsites/update-latest3

Update integration with latest main

devpatil7 committed:
Merge branch 'main' into merge-latest3

devpatil7 committed:
Merge pull request #596 from hlxsites/merge-latest3

Merge latest main

devpatil7 committed:
Fix PDP gallery zoom (#598)

* fix zoom

* update to beta

fnhipster committed:
Reorder main initialization in `scripts.js` (#595) (#611)

* Moved the calls to `decorateMain` and `applyTemplates` to occur after commerce initialization, ensuring that the main content is properly decorated and templates are applied only after commerce-related functions are executed. This change improves the loading sequence.

iivvaannxx committed:
ACAP-1062: Fix AEM Assets Tests (#607)

sirugh committed:
Merge branch 'main' into integration

jcalcaben committed:
Merge pull request #615 from hlxsites/merge-latest4

merge main into integration

eliseacornejo committed:
USF-2464 (#609)

emartinpalomas committed:
ACAP-1060 Product Discovery dropin compatibility to AEM Assets (#608)

sirugh committed:
Merge branch 'main' into merge-latest5

sirugh committed:
chore: improvements from sf-builder for ue compat (#622)

miakobchuk committed:
chore: remove default content from product-details block (#623)

Co-authored-by: Stephen <sirugh@users.noreply.github.com>

sirugh committed:
adds carousel and cards-list, as used by adobe store (#619)

Signed-off-by: Stephen Rugh <rugh@adobe.com>

sirugh committed:
Merge branch 'main' into merge-latest5

sirugh committed:
Merge pull request #621 from hlxsites/merge-latest5

Merges main into integration

sirugh committed:
Merge remote-tracking branch 'origin/main' into merge-latest

sirugh committed:
Merge pull request #624 from hlxsites/merge-latest

Merge latest

cctina516 committed:
USF 2382 - In cart flavor & size editing (#617)

* Create mini pdp component

* Implement mini pdp to edit button

* Add id to modal

* udpate layout fragment

* Add view all details button

* update layout

* Add styling for mini pdp

* Add error alert

* Add pre-selection

* Update success message in cart

* Add mini pdp to mini cart

* update edit button

* replace hard coded strings

* clean up

* udpate based on review

* fix styling

svera committed:
USF-2470: Wishlist - AXE tool reporting 1 critical and 1 moderate issue on wishlist page (#612)

iivvaannxx committed:
ACAP-1062: Fix AEM Assets Tests and Bump PDP to 1.2.1 (#618)

* fix: drafts page for cypress

* chore: update pdp back to 1.2.1

cctina516 committed:
Update SDK to 1.4.0-beta1 (#626)

* test SDK update with alpha package

* use beta

bl4de committed:
Update storefront-order to 1.3.0-beta1, storefront-auth to 2.0.4-alpha1

bl4de committed:
USF-2483: Recaptcha - When recaptcha V3 enabled - Place order failing

fnhipster committed:
Implement GitHub Action to block unauthorized AEM edits (#630)

* Implement GitHub Action to block unauthorized AEM edits by comparing local aem.js with upstream version

* Add playful comment to aem.js file

* fixed

* Enhance GitHub Action to enforce strict matching of scripts/aem.js with upstream version. Added conditional PR comment and failure exit for unauthorized modifications.

* test

* Refactor GitHub Action to simplify PR comment posting for aem.js changes. Removed direct API call and utilized GitHub CLI for commenting on PRs when unauthorized modifications are detected.

* Refactor GitHub Action to use a variable for PR comment body, improving readability and maintainability of the script that enforces strict matching of scripts/aem.js with the upstream version.

* Enhance GitHub Action to include environment variable for GitHub token and properly escape JSON for API calls when commenting on PRs. This improves the reliability of the comment posting process for aem.js changes.

* Remove unnecessary comment from aem.js file to improve code clarity.

cctina516 committed:
Integrate the undo feature from Cart (#629)

* use cart alpha

* add new config

* update to use latest alpha

* fix: make last item undo banner shown in empty cart and hide Gift Options in empty cart

* fix: fix cypress tests error in index.js

* use latest alpha

* add minicart config per QA

* set right column to display in empty cart

* fix minicart close

---------

Co-authored-by: yutingj <yutingj@adobe.com>

eliseacornejo committed:
update-wishlist (#632)

sivaschenko committed:
LYNX-920: Headers for personalization (#633)

anthoula committed:
COMOPT-862 Rec Add To Cart Click event (#628)

* COMOPT-862: Recs Add to Cart Click event

- add recs add to cart click event

* Comopt 862 fix2 (#634)

* find rec unit
* add Product id to request
* update drop-in
* fix productid
- bump alpha package

---------

Co-authored-by: benjaminkalk <benjaminkalk@gmail.com>
Co-authored-by: Benjamin Kalk <bkalk@adobe.com>

cctina516 committed:
Update cart to 1.5.0-beta1 (#638)

* use latest alpha

* use beta

aminakhyat committed:
ACAP-1061: Add AEM asset support for wishlist dropin (#643)

* Add AEM asset support for wishlist dropin

* skip wishlist test until support for ACCS

* remove getProductLink

* add tag to skip wishlist test in saas

keoko committed:
bumped drop-in auth 2.1.0-alpha2 (#642)

sirugh committed:
hide category facet on category page (#636)

Signed-off-by: Stephen Rugh <rugh@adobe.com>

loginesta committed:
Integrate enhanced Wishlist into boilerplate (#644)

* IIntegrate enhanced Wishlist into boilerplate

* fix cypress

* Change protocol for expectedOptions

* Use srcSetExpectedOptions

---------

Co-authored-by: Dev Patil <depatil@adobe.com>

andreituicu committed:
feat: Content Security Policy: strict-dynamic + (cached) nonce (#649)

* feat: Content Security Policy: strict-dynamic + (cached) nonce

* fix: 404 page not rendering

OscarMerino committed:
Update checkout dropin to v2.0.1-beta1 (#651)

loginesta committed:
Bump wishlist to version 2.0.0-beta1 (#653)

Co-authored-by: Dev Patil <40405790+devpatil7@users.noreply.github.com>

anthoula committed:
update @storefront-recommendations to v1.1.0-beta1 (#654)

cctina516 committed:
Update SDK to 1.4.0-beta3 and Cart to 1.5.0-beta2 (#648)

* test alpha

* use latest beta

* udpate cart version

---------

Co-authored-by: Dev Patil <40405790+devpatil7@users.noreply.github.com>

devpatil7 committed:
update dropins to latest (#655)

devpatil7 committed:
update to latest tools (#665)

devpatil7 committed:
Fix plp images (#661)

* fix plp images

* update to beta

* fix tests

devpatil7 committed:
skip flaky tests (#663)

* skip flaky tests

* skip tests

eliseacornejo committed:
unskip wishlist cypress saas tests (#660)

Co-authored-by: Dev Patil <40405790+devpatil7@users.noreply.github.com>

devpatil7 committed:
update auth (#666)

sirugh committed:
default analytics env should be testing (#670)

Signed-off-by: Stephen Rugh <rugh@adobe.com>

OscarMerino committed:
Server error does not appear in Safari browser (#669)

devpatil7 committed:
update tools (#672)

anthoula committed:
update @storefront-recommendations to v1.1.0-beta2 (#674)

cctina516 committed:
Update SDK to 1.4.0-beta6 and Cart to 1.5.0-beta3 (#675)

* use latest sdk

* use cart beta

devpatil7 committed:
Merge branch 'main' into suite-release3

iivvaannxx committed:
Unskip PaaS AEM Assets tests (#668)

* test: remove skipPaas tags

* test: fix inconsistencies

---------

Co-authored-by: Dev Patil <40405790+devpatil7@users.noreply.github.com>

devpatil7 committed:
update dropins to latest (#676)

* update dropins to latest

* update

* Upgrade Checkout dropin to v2.0.1

* update pdp version

---------

Co-authored-by: merino <merino@adobe.com>

devpatil7 committed:
update root package version (#679)

devpatil7 committed:
Merge pull request #658 from hlxsites/suite-release3

Suite release3

devpatil7 committed:
Merge pull request #680 from hlxsites/merge-latest

Merge latest main into integration

cctina516 committed:
Update Cart to 1.5.1-beta1  (#684)

* test alpha

* use beta

* skip failing flaky tests

---------

Co-authored-by: Dev Patil <depatil@adobe.com>

sirugh committed:
dont use pointer on non-interactive elements (#681)

miakobchuk committed:
support for metadata/schema - lowercase product sku in url (#406)

* refactor into single commit over integration and moves files around using new structure. 5d5d382f2861cb37e3b8a93329634407b5723980 for pre-refactor state

Signed-off-by: Stephen Rugh <rugh@adobe.com>

* dont use priceText if undefined

Signed-off-by: Stephen Rugh <rugh@adobe.com>

* fix price fetching

Signed-off-by: Stephen Rugh <rugh@adobe.com>

* rename checkSSGPage to isSsgPage

Signed-off-by: Stephen Rugh <rugh@adobe.com>

* remove pointer from all nav - just links should have pointer

Signed-off-by: Stephen Rugh <rugh@adobe.com>

* revert persisturl change

Signed-off-by: Stephen Rugh <rugh@adobe.com>

* Revert "remove pointer from all nav - just links should have pointer"

This reverts commit 2fea6862d5e08ad1a482382120f8405b440aa18c.

* only apply jsonld and meta tags if not prerendered

Signed-off-by: Stephen Rugh <rugh@adobe.com>

* Update scripts/commerce.js

* remove hydration logic:

Signed-off-by: Stephen Rugh <rugh@adobe.com>

* infer preload main image

Signed-off-by: Stephen Rugh <rugh@adobe.com>

* use aem fn for getMetadata

Signed-off-by: Stephen Rugh <rugh@adobe.com>

* helix only maps metadata for lowercase paths, but sku needs actual for client side requests, otherwise the page wont know how to transform the sku from the url

Signed-off-by: Stephen Rugh <rugh@adobe.com>

* force lowercase sku in plp

Signed-off-by: Stephen Rugh <rugh@adobe.com>

* utility function for product link creation

Signed-off-by: Stephen Rugh <rugh@adobe.com>

* make image extraction more robust

Signed-off-by: Stephen Rugh <rugh@adobe.com>

* fix lint

Signed-off-by: Stephen Rugh <rugh@adobe.com>

* generate JSON and provide upload instructions for DA

Signed-off-by: Stephen Rugh <rugh@adobe.com>

* use lowercase products urls

Signed-off-by: Stephen Rugh <rugh@adobe.com>

* running cicd

* avoid false positive for isprerendered check incase other script adds unrelated jsonld

Signed-off-by: Stephen Rugh <rugh@adobe.com>

* metadata readme

Signed-off-by: Stephen Rugh <rugh@adobe.com>

* some  missed lowercasings, incorrect urlKey

Signed-off-by: Stephen Rugh <rugh@adobe.com>

* add instructions for generating metadata in cypress

Signed-off-by: Stephen Rugh <rugh@adobe.com>

---------

Signed-off-by: Stephen Rugh <rugh@adobe.com>
Co-authored-by: Stephen Rugh <rugh@adobe.com>
Co-authored-by: Stephen <sirugh@users.noreply.github.com>

fnhipster committed:
Product Discovery v2 (#671)

* alpha package

* Update storefront-product-discovery to version 2.0.0-alpha002, add yarn.lock file, and enhance search functionality in header component.

* Update storefront-product-discovery to version 2.0.0-alpha003, bump tools to 1.4.1-alpha008, and enhance search functionality with pagination and improved event handling in product list page. Remove yarn.lock file.

* Enhance filter parameter handling in product list page by decoding URL-encoded values and supporting single value filters. This improves the robustness of the getFilterFromParams function.

* Refactor product list page search functionality to differentiate between category and search pages. Implement category filtering for category pages and adjust filter handling accordingly. Update facets rendering to anchor to the root category.

* Refactor product list page to clarify search request handling on block load, improving code readability and maintainability.

* fix linting

* Update storefront-product-discovery to version 2.0.0-alpha004, reflecting changes in package-lock.json and package.json. Enhance Facet component to include an optional 'last' prop for improved layout handling in facets rendering. Adjust SortBy component to ensure required prop is set for Picker component.

* Implement empty state styling for product list page and enhance search result handling. Add CSS rules to manage the display of elements when no products are found, and update JavaScript to reflect filter counts dynamically in the UI.

* Enhance header search result styling with new CSS rules for product item cards and skeletons. Introduce grid layout for better display and adjust skeleton item dimensions for improved visual consistency.

* lint

* Enhance pagination functionality in product list page by adding smooth scroll to top on page change. This improves user experience during navigation.

* Refactor pagination on product list page to simplify onPageChange handler. This change maintains smooth scrolling to the top while improving code clarity.

* Update storefront-product-discovery to version 2.0.0-alpha005, reflecting changes in package-lock.json and package.json. Refactor search handling to improve clarity and maintainability, including updates to the SortBy component and enhancements to the search API for better metadata management.

* lint

* Update @dropins/tools to version 1.4.0-beta5 in package.json and package-lock.json. Refactor header CSS for product item cards to streamline layout and improve styling consistency. Adjust product list page CSS for pagination layout. Enhance JavaScript in header to expose events globally for better event handling.

* Remove global exposure of events in header JavaScript to improve encapsulation and maintainability.

* Update storefront-product-discovery to version 2.0.0-alpha006. Refactor search handling in Cypress tests to improve element selectors and enhance test reliability. Adjust CSS for product discovery components to streamline layout and improve styling consistency.

* Fixed Cypress tests. Normalized border in Product Item Card.

* fix sort labels

* Refactor empty state layout and add error handling for product search

- Updated CSS for the empty state of the product list page to change grid layout.
- Added error handling in JavaScript for product search functionality to log errors in case of failure.

* Update storefront-product-discovery to version 2.0.0-alpha008

* Removed `rootCategory` from Facets container. Not longer used.

- Updated the rendering of the Facets component to remove the rootCategory prop, simplifying the implementation.

* Update storefront-product-discovery to version 2.0.0-alpha010

* Update storefront-product-discovery to version 2.0.0-alpha11 and remove unused rootCategory prop from Facets component. Simplified Facets rendering by eliminating the rootCategory parameter.

* skip aem tests

* Update Filters button label to use dynamic value from AEM

* Fixed i18n

* fixed label

* version bump

---------

Co-authored-by: Dev Patil <40405790+devpatil7@users.noreply.github.com>
Co-authored-by: Dev Patil <depatil@adobe.com>

sirugh committed:
Merge remote-tracking branch 'origin/main' into merge-patch

devpatil7 committed:
Merge pull request #687 from hlxsites/merge-patch

chore: main->int

sirugh committed:
fix metadata for multistore and jsonld path (#686)

Signed-off-by: Stephen Rugh <rugh@adobe.com>
Co-authored-by: Dev Patil <40405790+devpatil7@users.noreply.github.com>

fnhipster committed:
[USF-2728] Fix placeholder data loss bug  (#689)

* Fix placeholder data loss bug in fetchPlaceholders function - return accumulated data instead of latest fetch only

* fix cypress

---------

Co-authored-by: Dev Patil <depatil@adobe.com>

devpatil7 committed:
Merge branch 'main' into merge-fix

devpatil7 committed:
update product path

devpatil7 committed:
Merge pull request #693 from hlxsites/merge-fix

Merge main to integration

cctina516 committed:
Update SDK to 1.5.0-beta1 (#694)

* test alpha

* use beta

sirugh committed:
clean up readme (#695)

Signed-off-by: Stephen Rugh <rugh@adobe.com>

sirugh committed:
feature: fallback to "commerce-endpoint" if only one is defined. (#700)

* fallback to commerce-endpoint if core-endpoint not defined

Signed-off-by: Stephen Rugh <rugh@adobe.com>

* hardcoded example config of single endpoint

Signed-off-by: Stephen Rugh <rugh@adobe.com>

* Delete config.json

* Update blocks/commerce-checkout/commerce-checkout.js

---------

Signed-off-by: Stephen Rugh <rugh@adobe.com>

fnhipster committed:
Bug Fix: PDP state overwritten when multiple PDPs are loaded on the same page (#696)

* Update storefront-pdp to version 1.3.1-alpha001 and enhance API functionality

This update aims to enhance the modularity and usability of the storefront PDP components.

* Update storefront-pdp to version 2.0.0-alpha100 and tools to version 1.4.1-alpha100; refactor rendering in commerce-mini-pdp for improved readability

* Update storefront-pdp to version 1.3.1-alpha101 and tools to version 1.4.1-alpha101 in package.json.

* bump

* Update storefront-pdp to version 1.3.1-alpha102 and tools to version 1.4.1-alpha102

* Refactor fallbackData structure in commerce-mini-pdp for improved clarity and maintainability

* Update storefront-pdp to version 1.3.1-alpha103 and tools to version 1.4.1-alpha103; refactor ProductOptions component for improved data handling

* update tools and pdp versions

---------

Co-authored-by: Dev Patil <depatil@adobe.com>

cctina516 committed:
Update SDK to 1.5.0-beta2 (#705)

* test alpha

* use beta

devpatil7 committed:
Merge branch 'main' into merge-main

devpatil7 committed:
resolve merge conflicts from main

Commit:
INTERNAL: Inject payment services lang definitions (#704)

devpatil7 committed:
Merge branch 'integration' into merge-main

devpatil7 committed:
Merge pull request #706 from hlxsites/merge-main

Merge Patch release to integratrion

devpatil7 committed:
Add test coverage and integrate percy visual regression (#600)

* fix zoom

* Integrate Percy

* update

* update percy scope

* update

* local cypress launch script

* search test

* update product search

* update

* update my account test

* update

* update

* update

* update

* update

* update tests

* update

* fix tests

* update

* update flow

* update

* update

* update

* update

* update

* update

* update

* update

* update

* Debug mobile percy  (#712)

* check updates

* debug

* update

* update

* update

* update

* update

* update

* update

* update

* update

* update

* update to include all screens

* update

* update

sirugh committed:
chore: if config is missing or invalid, show a relevant error view (#714)

Signed-off-by: Stephen Rugh <rugh@adobe.com>

eliseacornejo committed:
update wishlis version (#713)

cctina516 committed:
Update Cart to 1.5.1-beta2 (#716)

* test alpha

* use beta

---------

Co-authored-by: Dev Patil <40405790+devpatil7@users.noreply.github.com>

fnhipster committed:
Update ACDL Scripts (#711)

* Update Adobe Client Data Layer to version 3.0.0 and remove ajv2020 validation scripts

- Updated the dependency for @adobe/adobe-client-data-layer to version 3.0.0 in package.json and package-lock.json.
- Added new files for the Adobe Client Data Layer in the postinstall script.
- Removed ajv2020.min.js and validate.js files as part of the cleanup.
- Adjusted commerce.js to remove validation initialization for the Adobe Client Data Layer.

* Update Adobe Client Data Layer to version 3.0.1 in package.json and package-lock.json; update minified script and source map for compatibility.

---------

Co-authored-by: Dev Patil <40405790+devpatil7@users.noreply.github.com>

cctina516 committed:
Fix Update cart API - USF 2503 (#702)

* when only updating quantity

* use cart alpha

* fix lint

* commit after install

---------

Co-authored-by: Dev Patil <40405790+devpatil7@users.noreply.github.com>

virfaure committed:
Disable Fastlane in checkout

virfaure committed:
Merge pull request #720 from hlxsites/PAY-6255

PAY-6255 - Disable Fastlane in checkout

buuhuu committed:
Merge remote-tracking branch 'upstream/main' into sept-2025

sirugh commented on blocks/commerce-cart/commerce-cart.js:
```diff
@@ -44,13 +47,16 @@ export default async function decorate(block) {
     'start-shopping-url': startShoppingURL = '',
     'checkout-url': checkoutURL = '',
     'enable-updating-product': enableUpdatingProduct = 'false',
+    'undo-remove-item': undo = 'false',
```
new block config.

sirugh commented on blocks/commerce-mini-cart/commerce-mini-cart.js:
```diff
@@ -15,6 +25,7 @@ export default async function decorate(block) {
     'cart-url': cartURL = '',
     'checkout-url': checkoutURL = '',
     'enable-updating-product': enableUpdatingProduct = 'false',
+    'undo-remove-item': undo = 'false',
```
new block config

sirugh commented on scripts/components/commerce-mini-pdp/commerce-mini-pdp.js:
```diff
@@ -0,0 +1,318 @@
+import { events } from '@dropins/tools/event-bus.js';
```
This is not actually a block - we are going to move it to scripts or library later in commerce boilerplate.

anthoula committed:
update @storefront-recommendations to v1.1.1-beta1 (#717)

* update @storefront-recommendations to v1.1.1-beta1

sirugh committed:
moves the mini-pdp to a components folder as it is not a block (#722)

* moves the mini-pdp to a components folder as it is not a block

Signed-off-by: Stephen Rugh <rugh@adobe.com>

* missed path

Signed-off-by: Stephen Rugh <rugh@adobe.com>

---------

Signed-off-by: Stephen Rugh <rugh@adobe.com>
Co-authored-by: Dev Patil <40405790+devpatil7@users.noreply.github.com>

devpatil7 committed:
Update Percy build trigger (#718)

cctina516 committed:
Update SDK to 1.5.0-beta3 (#727)

* use alpha

* use beta

sirugh committed:
[docs] Add Commerce Block README files (#728)

* generated readmes for each commerce block

Signed-off-by: Stephen Rugh <rugh@adobe.com>

* uses comments to hide readme sections that do not have useful content

Signed-off-by: Stephen Rugh <rugh@adobe.com>

* add comments around each excluded markdwon section

Signed-off-by: Stephen Rugh <rugh@adobe.com>

* Add automation for block README checks

* fixed commerce-checkout README: one-page checkout, duplicated order/placed

* move husky precommit script

Signed-off-by: Stephen Rugh <rugh@adobe.com>

* prevent commit without verify

Signed-off-by: Stephen Rugh <rugh@adobe.com>

---------

Signed-off-by: Stephen Rugh <rugh@adobe.com>
Co-authored-by: Natxo Cabré <cabre@adobe.com>

devpatil7 committed:
Upgrade PDP and tests for PDP and Cart (#729)

* Update Percy build trigger

* update

* update to latest PDP

* Include gift card container

* update test

* Cover giftcard

* Update storefront-pdp version

* Update pdp version

* Update pdp version to 1.3.4

* update

* check qty

---------

Co-authored-by: Stefan Perju <sperju@adobe.com>
Co-authored-by: Stephen <sirugh@users.noreply.github.com>

cctina516 committed:
Update Cart to 1.5.1-beta3 (#734)

* use alpha

* use beta

eliseacornejo committed:
USF-2822: update wihslist readme (#737)

Commit:
PAY-6280: Upgrade NPM deps to avoid supply-chain vulnerability (#741)

Upgrade NPM packages beyond versions vulnerable to the
recent NPM supply chain attack to rule out the possibility
of upgrading to a vulnerable version.

- Upgrade 'chalk'       5.4.1 -> 5.6.2  (5.6.1 vulnerable)
- Upgrade 'ansi-regex'  6.1.0 -> 6.2.2  (6.2.1 vulnerable)
- Upgrade 'ansi-styles' 6.2.1 -> 6.2.3  (6.2.2 vulnerable)
- Upgrade 'strip-ansi'  7.1.0 -> 7.1.2  (7.1.1 vulnerable)
- Upgrade 'wrap-ansi'   9.0.0 -> 9.0.2  (9.0.1 vulnerable)
- Upgrade 'error-ex'    1.3.2 -> 1.3.4  (1.3.3 vulnerable)

anthoula committed:
update @storefront-recommendations to v1.1.1-beta2 (#742)

sirugh committed:
add commerce-assets-enabled: false to demo-config

Signed-off-by: Stephen Rugh <rugh@adobe.com>

cctina516 committed:
use beta (#746)

cctina516 committed:
Update Cart to 1.5.1-beta4 (#758)

* test alpha

* use beta

sirugh committed:
replace default root category value with a placeholder so that users know to update this

Signed-off-by: Stephen Rugh <rugh@adobe.com>

sirugh committed:
chore: Check meta sku first (#761)

* check meta sku first, fall back to url sku

* no export for private

Signed-off-by: Stephen Rugh <rugh@adobe.com>

---------

Signed-off-by: Stephen Rugh <rugh@adobe.com>

sirugh committed:
remove outdated picker (#762)

Signed-off-by: Stephen Rugh <rugh@adobe.com>

sirugh committed:
remove references to hardcoded customer group header (#764)

Signed-off-by: Stephen Rugh <rugh@adobe.com>

aminakhyat committed:
Fix AEM Assets decimal dimension compatibility issue (#767)

Co-authored-by: Dev Patil <40405790+devpatil7@users.noreply.github.com>

mcocapiza committed:
Upgrade integration branch to include Checkout release v2.1.0 (#766)

* Upgrade integration branch to include Checkout release v2.1.0

* Increased temporarily the time to place an order

* Rollback the time to 5000

* place-order tests fails constantly in PaaS, but not in SaaS

* removed extra adobeLayer expect

* update account and order packages to latest

---------

Co-authored-by: Miquel Angel Coca Piza <cocapiza@Miquels-MacBook-Pro.local>
Co-authored-by: Natxo Cabré <cabre@adobe.com>
Co-authored-by: Natxo Cabré <natxo.cabre@gmail.com>
Co-authored-by: Dev Patil <depatil@adobe.com>

devpatil7 committed:
upgrade to latest auth (#770)

devpatil7 committed:
update dropin tools (#771)

devpatil7 committed:
Merge branch 'main' into syncmain

fnhipster committed:
removed duplicate code

devpatil7 committed:
Merge pull request #773 from hlxsites/syncmain

Sync with main

aminakhyat committed:
ACAP-1130 (#760)

* Fix preloadMiddleware when AEM assets integration is enabled

* Fix preloadMiddleware for aem assets

* update tools

---------

Co-authored-by: Dev Patil <40405790+devpatil7@users.noreply.github.com>
Co-authored-by: Dev Patil <depatil@adobe.com>

devpatil7 committed:
update search dropin (#776)

* update search dropin

* update to beta

keoko committed:
disabled fastlane payment method (#777)

devpatil7 committed:
update tools and pdp to latest (#779)

devpatil7 committed:
upgrade dropin packages to latest production (#781)

* upgrade dropin packages

* update

devpatil7 committed:
update packages for Prex and Wishlist (#784)

mcocapiza committed:
New Checkout release version v2.1.0 (#786)

Co-authored-by: Miquel Angel Coca Piza <cocapiza@Miquels-MacBook-Pro.local>

devpatil7 committed:
update root package version (#788)

devpatil7 committed:
Merge pull request #775 from hlxsites/suite-release4

Suite release 4

msagolj committed:
Merge tag 'october-2025' into sept-2025

msagolj committed:
update to @dropins/tools@1.5.1

msagolj committed:
placeholder update list

msagolj committed:
fix comparsion md

msagolj committed:
update placeholder-comparison

msagolj committed:
add match function for EDS to initializeCommerce()

msagolj committed:
readme add link to placeholder comparison

msagolj committed:
fix lint issue

msagolj committed:
remove folder mapping

msagolj committed:
add sku metadata for pdp pages

msagolj committed:
update readme for product pages setup

buuhuu committed:
feat: add undo-remove-item to cart and mini cart

buuhuu committed:
fix: pin dropin/tools to patch releases

buuhuu committed:
docs: updated readme
